lib/provider.js

```diff
@@ -33,14 +33,18 @@ module.exports = {
 
   getSuggestions({editor, bufferPosition, scopeDescriptor}) {
     const prefix = getPrefix(editor, bufferPosition);
+    const filePath = editor.getPath();
 
     if (shouldPreventAutocomplete(prefix)) {
       return null;
     }
 
+    if (filePath === undefined) {
```

Comment:
Sure thing, oftentimes good to be explicit :) In this case I think it's safe to assume all falsy values are bad filePaths, so let's keep the `!filePath`.